rtrlib/rtr/rtr.c

```diff
@@ -245,19 +246,19 @@ void rtr_stop(struct rtr_socket *rtr_socket)
     RTR_DBG1("Socket shut down");
 }
 
```

Comment:
I'm not sure if there is a good usecase to use rtr-sockets directly without the rtrmgr, but If we want to keep the option to use the rtr-sockets without the rtr-mgr, all of the the functions except the `static` ones needs to be exported.

Reply:
Using rtr_sockets without the manager would require initializing the tables yourself, which you cannot do anymore because neither `pfx_table_init` nor `spki_table_init` are exported.

Reply:
Yes, that would require to export more other symbols, probably most of the ones that are defined in the ".h" files.
Initially rtrmgr was only an additional wrapper that you could use if you want to use multiple rtr-sockets.
Depends on what the goal is, if rtrmgr should be the main interface to use the library.

Reply:
I think providing rtr_mgr as the only interface should be goal at least long term.
It would make developing rtrlib much easier.